app/_src/guides/restrict-permissions-to-selected-namespaces-on-kubernetes.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Restrict permissions to selected namespaces on Kubernetes
+content_type: tutorial
+---
+
+{% capture docs %}/docs/{{ page.release }}{% endcapture %}
+{% assign Kuma = site.mesh_product_name %}
+{% assign kuma = site.mesh_install_archive_name | default: "kuma" %}
+{% assign kuma-control-plane-workloads = kuma | append: "-control-plane-workloads" %}
+
+By default, {{ Kuma }} deployed on Kubernetes has permissions to observe and react to events from resources across the entire cluster. While this behavior simplifies initial setup and testing, it might be too permissive for production environments. Limiting {{ Kuma }}'s access to only necessary namespaces helps enhance security and prevents potential impact on unrelated applications.
+
+Starting from version 2.11, you can restrict the permissions granted to the {{ Kuma }} control plane. This guide explains how to limit {{ Kuma }} to specific namespaces, giving you greater control over security and resource management.
+
+## Prerequisites
+
+Before you start, ensure you have:
+
+* A clean, running, and accessible Kubernetes cluster
+* The `kubectl` command-line tool installed and configured
+
+## Restrict {{ Kuma }} to selected namespaces
+
+This section shows you how to restrict {{ Kuma }}'s permissions to selected namespaces, verify that restrictions are correctly applied, and update permissions if needed.
+
+### Step 1: Create and label the initial namespace
+
+```bash
+kubectl create namespace first-namespace
+kubectl label namespace first-namespace kuma.io/sidecar-injection=enabled
+```
+
+### Step 2: Install {{ Kuma }} restricted to the first namespace
+
+{% cpinstall namespaceAllowList %}
+namespaceAllowList={first-namespace}
+{% endcpinstall %}
+
+### Step 3: Deploy a test workload in the allowed namespace
+
+```bash
+kubectl run nginx --image=nginx --port=80 --namespace first-namespace
+```
+
+### Step 4: Verify the first namespace is working
+
+Check that the control plane is managing the workload in `first-namespace`:
+
+```bash
+kubectl get dataplanes --namespace first-namespace
+kubectl get pods --namespace first-namespace
+kubectl get rolebindings --namespace first-namespace
+```
+
+You should see:
+
+* A `Dataplane` resource listed
+* The pod showing two containers (`2/2`), one for `nginx` and one for the sidecar
+* A `RoleBinding` named `{{ kuma-control-plane-workloads }}` that grants the control plane elevated access to manage resources within the namespace
```

Comment:
Technically because the pod has a defined name now you could just show the output

Reply:
addressed